### DISCUSSION TITLE
Multiple storage accounts

### DISCUSSION THREAD

**Discussion by alexwilcoxson-rel** [databendlabs/databend#9726] [Q&A] [answered]
Hi I am evaluating databend for our use case. We have multiple Azure storage accounts for different clients' data. Can a single databend instance have separate databases that are managed in separate storage accounts?

**Comment by sundy-li** [comment, marked as answer]
We support external location in fuse engine, see https://databend.rs/doc/sql-commands/ddl/table/ddl-create-table#create-table--external_location

**Reply by alexwilcoxson-rel** [reply]
Thanks! That looks like it only supports AWS at this time. Also are there disadvantages with external tables compared to within FUSE?

**Reply by sundy-li** [reply]
Yes, you can have your custom `UriLocation`.

https://github.com/datafuselabs/databend/blob/993421e2e018f7d8086010c79d12a2a3a363950a/src/query/sql/src/planner/binder/location.rs#L268-L332


@ariesdevil  need to improve the documents.
